graders/r/Dockerfile

```
@@ -31,12 +31,13 @@ ENV PYTHONIOENCODING=UTF-8
 # tidyverse'. Switching to using the CRAN package names may make it easier to other R
 # users to adapt and adjust the package list.  We also install sqlite3 to support the
 # SQL lectures. (Adding a duckdb binary may be an option too.)
-RUN apt update \
-    && apt install -y curl \
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends curl \
                       git \
                       pandoc \
                       sqlite3 \
                       sudo \
```

Nit, for consistency:

```suggestion
    && apt-get install -y --no-install-recommends curl git pandoc sqlite3 sudo \
```

I also notice that we're inconsistent with `&&` appearing at the start of lines, or at the end of them (this comment applies to the repo as a whole). Could be nice to be more consistent there too.

---

Can we leave the consistency nits with `&&` for another PR? I don't feel particularly strong about it in either direction, and I don't want formatting and linting to get confused.